### PR TITLE
Generate SGs/nACLs for all resources

### PR DESCRIPTION
**Pull request by YairSlobodin1**

### CODE REVIEW

File: pkg/ir/unspecified.go [hunk 1]
```diff
@@ -7,32 +7,25 @@ package ir
...
 	"strings"
 )
 
 const (
-	commonWarningACL            = "The following subnets do not have required connections; "
-	warningUnspecifiedACL       = commonWarningACL + "no ACLs were generated for them: "
-	warningUnspecifiedSingleACL = commonWarningACL + "the generated ACL will block all traffic: "
-	warningUnspecifiedSG        = "The following endpoints do not have required connections; no SGs were generated for them: "
+	warningUnspecifiedACL = "The following subnets do not have required connections; the generated ACL will block all traffic: "
+	warningUnspecifiedSG  = "The following endpoints do not have required connections; the generated SGs will block all communication: "
```
Comment (zivnevo) [inline pkg/ir/unspecified.go:16]:
```suggestion
	warningUnspecifiedSG  = "The following endpoints do not have required connections; the generated SGs will block all traffic: "
```

File: pkg/io/tfio/acl.go [hunk 2]
```diff
@@ -88,13 +90,21 @@ func aclCollection(t *ir.ACLCollection, vpc string) *tf.ConfigFile {
...
 		Resources: acls,
 	}
 }
+
+func subnetCidr(acl *ir.ACL) *ipblock.IPBlock {
+	if len(acl.Internal) > 0 {
+		return acl.Internal[0].Target()
+	}
+	return acl.External[0].Target()
+}
```
Comment (zivnevo) [inline pkg/io/tfio/acl.go:110]: Don't we have a better way to extract the CIDR of a subnet?

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/io/tfio/acl.go (hunk 2 above)
Comment (YairSlobodin1) [reply pkg/io/tfio/acl.go:110]: not from `ir.ACL` struct

Comment (zivnevo) [reply pkg/io/tfio/acl.go:110]: does this information exist in `ir` at all?

File: pkg/ir/unspecified.go (hunk 1 above)
Comment (kyorav) [reply pkg/ir/unspecified.go:16]: I agree

File: pkg/io/tfio/acl.go (hunk 2 above)
Comment (YairSlobodin1) [reply pkg/io/tfio/acl.go:110]: yes, in `ir.ConfigDefs`. [link](https://github.com/np-guard/vpc-network-config-synthesis/blob/main/pkg/ir/spec.go#L65).

Comment (zivnevo) [reply pkg/io/tfio/acl.go:110]: ok. Let's hope it won't bite us in the future.

**Pull request merged**